packages/webcomponents/src/api/Payment.ts

```diff
@@ -284,3 +299,30 @@ export class PaymentBalanceTransaction implements IPaymentBalanceTransaction {
       : this.source_id;
   }
 }
+
+export interface IRefund {
+  id: string;
+  payment_id: string;
+  amount: number;
+  description: string | null;
+  reason: string | null;
+  status: string;
+  metadata: null;
+  created_at: string;
+  updated_at: string;
+}
+
+export interface IApplicationFee {
+  id: string;
+  amount: number;
+  currency: string;
+  created_at: string;
+  updated_at: string;
+}
+
+export interface FetchPaymentsResponseType {
```

Comment:
I am curious - what is the reason we created a new type interface for the Payment List response, instead of using `IApiResponse` or `IApiResponseCollection` from `Api.ts`? 

It contains the exact properties you have specified here. 🤔

Reply:
Hey, nice catch! @colganmeanor 

I mislooked that we had these Api response types already. It makes total sense to use them. I am changing this to use our existing response types. 👍